extensions/ql-vscode/package.json

```diff
@@ -180,6 +180,11 @@
           "default": false,
           "description": "Enable debug logging and tuple counting when running CodeQL queries. This information is useful for debugging query performance."
         },
+        "codeQL.runningQueries.maxPaths": {
+          "type": "integer",
+          "default": 4,
```

Comment:
Makes sense! Do you have an idea what counts as a sensible upper limit? I've put 256 for now, since that's the highest number I've seen "[in the wild](https://lgtm.com/projects/g/openjdk/jdk/alerts/?mode=list&lang=java)".

Reply:
I appreciate the data-driven decision. Much better than the random choice I was going to suggest!